types/index.d.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types='node' />
+
+import {
+  FastifyPluginCallback,
+} from 'fastify';
+
+declare module 'fastify' {
+  interface FastifyContextConfig {
+    throttle?: fastifyThrottle.FastifyThrottleOptions;
+  }
+}
+
+type FastifyThrottle = FastifyPluginCallback<fastifyThrottle.FastifyThrottleOptions>;
+
+declare namespace fastifyThrottle {
+
+  export interface FastifyThrottleOptions {
+    bps: number | ((elapsedTime: number, bytes: number) => number)
+    /**
+     * Throttle stream payloads
+     * @default true
+     */
+    streamPayloads?: boolean
```

Comment:
why not call it throttleStreamPayloads?

Reply:
it is about throttling payloads which are streams. not a general trigger for throttling or not